principles/coding-principles.md

```diff
@@ -89,13 +97,15 @@ When the work in the branch is complete create a pull request which can be revie
 When approving pull requests keep in mind the recommendations in the Coding Standards and ensure that any questionable code is challenged appropriately.
 
 ### 3. Delete branches once they are merged
-Once the PR is approved and you are merging back into the master branch ensure that the original branch is deleted. This prevents the repository becming littered with dead branches
+Once the PR is approved and you are merging back into the master branch ensure that the original branch is deleted. This prevents the repository becoming littered with dead branches
 
-## Technical Debt Principles
+## Technical Debt
 
 ### 1. Logged technical debt 
-
-Technical Debt should be logged (we curently use Jira to do this) and triaged taking into account the effort to pay it back, the consequences and the risk. Then, depending on the identified severity taken directly into the appropriate team's backlog for completion or pushed into the Single Backlog where a larger impact on delivery is identified.
+Technical Debt should be logged (we currently use Jira to do this) and triaged taking into account the effort to pay it back, the consequences and the risk. Then, depending on the identified severity taken directly into the appropriate team's backlog for completion or pushed into the Single Backlog where a larger impact on delivery is identified.
 
 ### 2. Ensure technical debt is communicated
 When the decision is made to accept technical debt ensure that the decision has been communicated to the relevant people so that it can be understood and prioritised.
```

Comment:
I think this should include a list of the 'appropriate people'. For example I think it needs to include you Dan B so that a team cannot simply say 'yeah but we've got loads of tech debt because we haven't been doing a good job for the last 6 months'.

Reply:
This document is more around the principles of what we should be doing. I want to keep it high level and without individuals being named etc. 

How we actually manage these principles I think should be handled elsewhere. Such as through clan confluence pages and direct clan interactions etc....

Reply:
Ok yeah so the people might not be the right thing but I think the roles should be listed. 

I'd be fine with that being on Confluence but then this would need a link to that page to allow people to find it easily (and more importantly have no excuse for NOT finding it!!)

Reply:
I'd prefer to link the Confluence pages to this site rather than the other way around.

As this page is public and Confluence is private I think that would be a better UX so that people without access to our Confluence wouldn't end up with dead-end links whilst looking at our principles.